modules/phase_field/examples/anisotropic_interfaces/GrandPotentialTwophaseAnisotropy.i

```diff
@@ -7,13 +7,16 @@
   xmax = 4
   ymin = -4
   ymax = 4
+  uniform_refine = 2
 []
 
 [GlobalParams]
   radius = 0.5
   int_width = 0.3
   x1 = 0
   y1 = 0
+  derivative_order = 2
+  enable_jit = false
```

Comment:
is this still necessary?

Reply:
Just to speed up a little.

Reply:
No: (running with 4 MPI processes)

```
JIT=false
real	1m49.247s
user	7m5.080s
sys	0m10.293s

JIT=true (1st run)
real	1m21.971s
user	5m17.654s
sys	0m9.571s

JIT=true (2nd run)
real	1m20.879s
user	5m14.455s
sys	0m8.708s
```

Runs **over 30% faster with JIT**, and the JIT overhead (difference between 1st and 2nd run, where the cached functions are used) is only one second. It does not make sense to disable JIT here!

Reply:
I see. Okay, I will enable it then.